core/sass/msg.scss

```diff
@@ -39,7 +39,7 @@
   -moz-border-radius:7px;
   -webkit-border-radius:7px;
   border-radius:7px;
-  font-size:1em;
+  font-size:10em;
```

Comment:
Oh! right... forgot to change that back. My bad... lol.

On Wed, Apr 15, 2015 at 9:57 PM, Dylan Hoffman <dylhoff@gmail.com> wrote:

> ???
>
>
> On Wed, Apr 15, 2015 at 9:56 PM, Ian Hunter <notifications@github.com>
> wrote:
>
>> In core/sass/msg.scss
>> <https://github.com/BYU-ODH/hlrdesk/pull/245#discussion_r28482450>:
>>
>> > @@ -39,7 +39,7 @@
>> >    -moz-border-radius:7px;
>> >    -webkit-border-radius:7px;
>> >    border-radius:7px;
>> > -  font-size:1em;
>> > +  font-size:10em;
>>
>> hmmm.... [image: :sunglasses:]
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/BYU-ODH/hlrdesk/pull/245/files#r28482450>.
>>
>
>